.circleci/config.yml

```diff
@@ -242,27 +242,26 @@ jobs:
       - run: swiftformat megazords components/*/ios --lint --swiftversion 4 --verbose
   Check Rust formatting:
     docker:
-      - image: circleci/rust:1.38
+      - image: circleci/rust:latest
     steps:
       - checkout
-      - setup-rust-toolchain:
-          rust-version: 1.38.0
+      - setup-rust-toolchain
       - run: rustup component add rustfmt
       - run: rustfmt --version
       - run: cargo fmt -- --check
   Lint Rust with clippy:
     docker:
-      - image: circleci/rust:1.38
+      - image: circleci/rust:latest
     steps:
-      - test-setup:
-          rust-version: 1.38.0
+      - test-setup
       - run: rustup component add clippy
       - run: cargo clippy --version
-      - run: cargo clippy --all --all-targets --all-features -- -D warnings
+      - run: cargo clippy --all --all-targets --all-features -- -D warnings -A clippy::redundant_clone
       # Run clippy without any features so that we still lint things that are
       # behind a `#[cfg(not(feature = "..."))]`, which will always be false with
       # --all-features.
-      - run: cargo clippy --all --all-targets --no-default-features -- -D warnings
```

Comment:
This started failing with Rust 1.40.0, I can open a follow-up to see if we can do anything about this.

Reply:
See https://github.com/mozilla/application-services/blob/ffb8f3cd87e24e0f8fdf4b48af10a6dd69bda48d/.circleci/config.yml#L148-L163, where we worked around this elsewhere.